_data/tutorials.yaml

```diff
@@ -484,3 +483,24 @@ masking-data:
     ruby: disabled
     scala: disabled
     swift: disabled
+
+anomaly-detection:
+  title: "Detecting abnormal events"
+  meta-description: "Detecting abnormal events"
+  slug: "/anomaly-detection"
+  question: "How do I find abnormal events?"
```

Comment:
@awalther28 is it possible that "abnormal events" might be too domain specific?  If so, maybe add a clause here with a few more words that generalizes this, because "abnormal events" might not be well understood or descriptive enough?